Week 04/Lecture 08/Assignment3/assignment3/src/main/java/com/lecture8/assignment3/config/AppConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.lecture8.assignment3.config;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+
+    
+    @Bean
+    @Qualifier("dataSource1")
+    public DataSource dataSource1() {
+        DataSourceBuilder dataSourceBuilder = DataSourceBuilder.create();
+        dataSourceBuilder.driverClassName("com.mysql.jdbc.Driver");
+        dataSourceBuilder.url("jdbc:mysql://localhost:3306/lecture8db1?useSSL=false&serverTimezone=Asia/Jakarta");
```

Comment:
All database infomation should be placed in properties file and using @Value to get.